packages/volto/src/server.jsx

```diff
@@ -224,9 +224,33 @@ server.get('/*', (req, res) => {
 
   loadOnServer({ store, location, routes, api })
     .then(() => {
+      const defaultLanguage =
+        store.getState().site?.data['plone.default_language'] || 'en';
+
+      const supportedLanguages =
+        store.getState().site?.data['plone.available_languages'] || 'en';
```

Comment:
Should we maybe default to what's in `config.settings`? I feel like this creates a breaking change otherwise for sites that maybe don't have those keys exported from `@site`? Honestly though I don't know if this is a scenario that is plausible.